opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/ConfigMerge.java

```diff
@@ -160,18 +161,18 @@ public static void main(String[] argv) {
         cfgNew.encodeObject(os);
     }
 
-    private static final void a_usage() {
-        System.err.println("Usage:");
-        System.err.println(name + " [-h] <config_file_base> <config_file_new>");
-        System.err.println();
-        System.err.println("OPTIONS:");
-        System.err.println("Help");
-        System.err.println("-?                   print this help message");
-        System.err.println("-h                   print this help message");
-        System.err.println();
+    private static final void a_usage(PrintStream out) {
+        out.println("Usage:");
+        out.println(name + " [-h] <config_file_base> <config_file_new>");
```

Comment:
Please add some comment saying that base is usually read-only configuration etc.